builtin/logical/pki/storage.go

```diff
@@ -27,6 +27,9 @@ const (
 
 	// Used as a quick sanity check for a reference id lookups...
 	uuidLength = 36
+
+	maxRolesToScanOnIssuerChange = 10000
```

Comment:
nit: maybe drop this to 50 or so? This could incur a lot of overhead. /cc @sgmiller